hydra/test_utils/launcher_common_tests.py

```diff
@@ -489,22 +502,25 @@ def test_custom_sweeper_run_workdir(
         expected_dir: str,
         task_launcher_cfg: DictConfig,
         extra_flags: List[str],
+        expected_working_dir: Optional[Path]
     ) -> None:
         overrides = extra_flags + overrides
         task_launcher_cfg = OmegaConf.create(task_launcher_cfg or {})  # type: ignore
         task_config = OmegaConf.create(task_config or {})  # type: ignore
         cfg = OmegaConf.merge(task_launcher_cfg, task_config)
         assert isinstance(cfg, DictConfig)
+        expected_outputs = str(expected_working_dir / expected_dir) if expected_working_dir else str(tmpdir / expected_dir)
         integration_test(
             tmpdir=tmpdir,
             task_config=cfg,
             overrides=overrides,
             prints="os.getcwd()",
-            expected_outputs=str(tmpdir / expected_dir),
+            expected_outputs=expected_outputs,
+            post_run_callback=self.post_run_callback(),
         )
 
     def test_get_orig_dir_multirun(
```

Comment:
It looks like you are changing the semantics of this test (it's testing the output working directory, and you are providing different working directory as expected output).

Please explain why.

Reply:
I think we've resolved this one. Pls let me know otherwise :)